src/fragments/lib/graphqlapi/js/working-with-files.mdx

```diff
@@ -1187,6 +1197,14 @@ await Promise.all(
 </Block>
 </BlockSwitcher>
 
+## Working with real-time data
```

Comment:
Seems mis-titled.

This section casually suggests realtime data as a solution to further minimize the effects of data sync issues on clients. But, the section is more generally about "data consistency" and/or "defensive programming".

Reply:
Will update